extensions/pyRevitTools.extension/checks/audit_all_check.py

```diff
@@ -0,0 +1,396 @@
+
+        # RVTLinks
+        if rvtlinks_elements_items:
+            generate_rvt_links_report(output, rvtlinks_docs, body_css)
+        output.self_destruct(180)
+    except Exception as e:
+        print(format_exc())
+        print(e)
+
+
+def generate_rvt_links_report(output, rvtlinks_docs, body_css):
+    output.print_md('# RVTLinks')
+    for rvtlink in rvtlinks_docs:
+        link_name = doc_name(rvtlink)
+        link_printed_name = clean_name(rvtlink)
+        output.print_md('## ' + link_printed_name)
+        output.print_md('___')
+        if rvtlink is None:
+            continue
+        output.print_md(link_name)
+        element_count = elements_count(rvtlink)
+        purgeable_elements_count = get_purgeable_count(rvtlink)
+        all_warnings_count, _, warnings_guid = warnings(rvtlink)
+        critical_warnings_count = critical_warnings(warnings_guid,CRITICAL_WARNINGS)
+        rvtlinks_elements_items, rvtlinks_count, type_status, rvtlinks_docs = rvtlinks_elements(rvtlink)
+        rvtlinks_unpinned = rvt_links_unpinned_count(rvtlinks_elements_items)
+        activated_analytical_model_elements_count = (analytical_model_activated_count(rvtlink))
+        rooms_count, unplaced_rooms_count, unbounded_rooms = rooms(rvtlink)
+        sheets_count, sheets_set = sheets(rvtlink)
+        views_count, views = views_bucket(rvtlink)
+        views_not_on_sheets = views_not_sheeted(sheets_set, views_count)
+        schedule_count, schedules_not_sheeted_count = schedules_count(rvtlink)
+        copied_views_count = copied_views(views)
+        view_templates_count = view_templates(rvtlink)
+        unused_view_templates_count = unused_view_templates(views)
+        all_filters_count, unused_view_filters_count = filters(rvtlink, views)
+        materials_count = materials(rvtlink)
+        line_patterns_count = line_patterns(rvtlink)
+        dwgs_count, linked_dwg_count = dwgs(rvtlink)
+        inplace_family_count, not_parametric_families_count, family_count = families(rvtlink)
+        imports_subcats_count = subcategories_imports(rvtlink)
+        generic_models_types_count = generic_models(rvtlink)
+        detail_components_count = details_components(rvtlink)
+        text_notes_types_count, text_notes_types_wf_count, text_bg_count = text_notes_types(rvtlink)
+        text_notes_count, text_notes_caps_count = text_notes_instances(rvtlink)
+        detail_groups_count, detail_groups_types_count = detail_groups(rvtlink)
+        model_group_count, model_group_type_count = groups(rvtlink)
+        reference_planes_count, unnamed_ref_planes_count = reference_planes(rvtlink)
+        detail_lines_count = detail_lines(rvtlink)
+        dim_types_count, dim_count, dim_overrides_count = count_dimension_types(rvtlink), count_dimensions(rvtlink), count_dimension_overrides(rvtlink)
+        revision_clouds_count = revisions_clouds(rvtlink)
+        links_data = ''
+        if rvtlinks_count != 0:

```

Comment:
same thing as the main function, just use 
```suggestion
        if rvtlinks_elements_items:
```